prep-javascript-functions/main.js

```diff
@@ -0,0 +1,53 @@
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+var addTwoNumbersResult = addTwoNumbers(2, 2);
+console.log('addTwoNumbers Exercise:', addTwoNumbersResult);
+
+function convertHoursToMinutes(hours) {
+  return hours * 60;
+}
+var convertHoursToMinutesResult = convertHoursToMinutes(2);
+console.log('convertHoursToMinutes Exercise:', convertHoursToMinutesResult);
+
+function getGreeting(name) {
+  return 'Hello World!';
+}
+var getGreetingResult = getGreeting('World');
```

Comment:
For this function we need to use our `name` parameter in the concatenation so that it will work with any second word and not just World